readthedocs/search/faceted_search.py

```diff
@@ -21,7 +16,11 @@
 
 class RTDFacetedSearch(FacetedSearch):
 
-    def __init__(self, user, **kwargs):
+    """Custom wrapper around FacetedSearch."""
+
+    operators = []
```

Comment:
Added it here to make it clear this is a custom attribute, not something from the original `FacetedSearch` class